docs/getting-started/dashboard_tutorial.md

```diff
@@ -0,0 +1,1788 @@
+                        name="email",
+                    ),
+                    rx.select(
+                        ["Male", "Female"],
+                        placeholder="male",
+                        name="gender",
+                    ),
+                    rx.flex(
+                        rx.dialog.close(
+                            rx.button(
+                                "Cancel",
+                                variant="soft",
+                                color_scheme="gray",
+                            ),
+                        ),
+                        rx.dialog.close(
+                            rx.button(
+                                "Submit", type="submit"
+                            ),
+                        ),
+                        spacing="3",
+                        justify="end",
+                    ),
+                    direction="column",
+                    spacing="4",
+                ),
+                on_submit=State.add_user,
+                reset_on_submit=False,
+            ),
+            max_width="450px",
+        ),
+    )
+
+def graph():
+    return rx.recharts.bar_chart(
+        rx.recharts.bar(
+            data_key="value",
+            stroke=rx.color("accent", 9),
+            fill=rx.color("accent", 8),
+        ),
+        rx.recharts.x_axis(data_key="name"),
+        rx.recharts.y_axis(),
+        data=State.users_for_graph,
+        width="100%",
+        height=250,
+    )
+
+def index() -> rx.Component:
+    return rx.vstack(
+        add_customer_button(),
+        rx.table.root(
+            rx.table.header(
+                rx.table.row(
+                    rx.table.column_header_cell("Name"),
+                    rx.table.column_header_cell("Email"),
+                    rx.table.column_header_cell("Gender"),
+                ),
+            ),
+            rx.table.body(
+                rx.foreach(
+                    State.users, show_user
+                ),
+            ),
+            variant="surface",
+            size="3",
+            width="100%",
+        ),
+        graph(),
+        align="center",
+        width="100%",
+    )
+
+
+app = rx.App(
+    theme=rx.theme(
+        radius="full", accent_color="grass"
+    ),
+)
+
+app.add_page(
+    index,
+    title="Customer Data App",
+    description="A simple app to manage customer data.",
+    on_load=State.transform_data,
+)
+```
+
+And that is it for your first dashboard tutorial. In this tutorial we have created 
+
+- a table to display user data
+- a form to add new users to the table
+- a dialog to showcase the form
+- a graph to visualize the user data
+
+In addition to the above we have we have 
+
+- explored state to allow you to show dynamic data that changes over time
+- explored events to allow you to make your app interactive and respond to user actions
+- added styling to the app to make it look better
+
+
+
+## Advanced Section (Hooking this up to a Database)
```

Comment:
This would be quite nice, and can probably be explained concisely enough.

Reply:
will add in follow up